README.md

```diff
@@ -1,2 +1,4 @@
+[![Build](https://github.com/siriak/MoneyManager/workflows/Build/badge.svg)](https://github.com/siriak/MoneyManager/actions)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Could not find module `remark-lint-no-unneeded-full-reference-image`](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931948)